gbt/main_clicks.ipynb

```diff
@@ -324,8 +324,8 @@
    "id": "3789bcfb",
    "metadata": {
     "ExecuteTime": {
-     "end_time": "2021-11-19T22:09:00.172234Z",
-     "start_time": "2021-11-19T22:09:00.168240Z"
+     "end_time": "2022-01-14T17:53:52.639607Z",
+     "start_time": "2022-01-14T17:53:42.891Z"
```

Comment:
can we avoid committing notebooks with outputs ?

Reply:
Cleaned in commit [8ed9799](https://github.com/criteo-research/criteo-privacy-preserving-ml-competition/pull/1/commits/8ed979970e0ce0acd03cc5782b109a86af69f0a0)